moveit_ros/planning/planning_request_adapter_plugins/src/fix_start_state_collision.cpp

```diff
@@ -147,17 +145,16 @@ class FixStartStateCollision : public planning_request_adapter::PlanningRequestA
             added_index++;
           added_path_index.push_back(0);
         }
-        return moveit_code;
+        return solved;
       }
       else
       {
         RCLCPP_WARN(LOGGER,
                     "Unable to find a valid state nearby the start state (using jiggle fraction of %lf and %u sampling "
-                    "attempts).",
+                    "attempts). Passing the original planning request to the planner.",
                     jiggle_fraction_, sampling_attempts_);
         res.error_code_.val = moveit_msgs::msg::MoveItErrorCodes::START_STATE_IN_COLLISION;
-        return moveit::core::MoveItErrorCode(moveit_msgs::msg::MoveItErrorCodes::START_STATE_IN_COLLISION);
-        ;
+        return false;  // skip remaining planners
```

Comment:
```suggestion
        return false;  // skip remaining planning adapters
```

Reply:
I considered that before, but not only adapters are skipped but the main planner as well. As both are named with the variable `planner` in source, I decided for planners.

Reply:
```suggestion
        return false;  // skip remaining planners & adapters
```